.envs/.local/.django

```diff
@@ -2,3 +2,5 @@
 # ------------------------------------------------------------------------------
 USE_DOCKER=yes
 IPYTHONDIR=/app/.ipython
+
+DJANGO_SETTINGS_MODULE=config.settings.local-umkc
```

Comment:
See the link below for how to configure the environment. You should create a separate environment for local-umkc and production-umkc, and specify to use that environment in local-umkc.yml and production-umkc.yml

[https://cookiecutter-django.readthedocs.io/en/latest/developing-locally-docker.html?highlight=environment#configuring-the-environment
](https://cookiecutter-django.readthedocs.io/en/latest/developing-locally-docker.html?highlight=environment#configuring-the-environment
)

Reply:
I'd also try to look if there's some way inherit environment files or .yml files

Reply:
The goal is that you only have to run docker-compose -f local-umkc.yml up to start a local osler instance with umkc settings

Reply:
> I'd also try to look if there's some way inherit environment files or .yml files

seems there's no native way to inherit yaml files. There are some work arounds but probably not worth messing around with

Reply:
Don't love the solution I came up with since the umkc.yml files are just copy/pasted versions of the production.yml and local.yml files with 1 line added for the umkc specific env file. If you can find a better implementation, I'd be happy to change it around